hack/run-machine-controller.sh

```diff
@@ -18,8 +18,8 @@ set -e
 
 make -C $(dirname $0)/.. machine-controller
 $(dirname $0)/../machine-controller \
-  -kubeconfig=$(dirname $0)/../.kubeconfig \
-  -worker-count=50 \
+  -kubeconfig=$HOME/.kube/config \
```

Comment:
Please remove the changes made to this script, we use it when running e2e tests locally and that will break with these changes: https://github.com/kubermatic/machine-controller#end-to-end

Also one worker will slow everything down severly when managing more than one machine ;)

Other than that everything seems to work fine, thanks for tackling this!

/test all